.github/workflows/lint_markdown.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 The StableHLO Authors.
+
+name: Markdown Lint
```

Comment:
Can we remove this from the PR? We currently have GH set up to require that all checks run in order to merge, I'm worried this not running will prevent that.

Reply:
Sounds good. Done.